Frameworks/Account/AccountTests/Feedly/FeedlySendArticleStatusesOperationTests.swift

```diff
@@ -74,15 +75,26 @@ class FeedlySendArticleStatusesOperationTests: XCTestCase {
 		
 		waitForExpectations(timeout: 2)
 		
-		XCTAssertEqual(container.database.selectPendingCount(), 0)
+		let selectPendingCountExpectation = expectation(description: "Did Select Pending Count")
+		container.database.selectPendingCount { result in
+			do {
+				let statusCount = try result.get()
+				XCTAssertEqual(statusCount, 0)
+				selectPendingCountExpectation.fulfill()
+			} catch let e {
+				XCTFail("Error unwrapping database result: \(e)")
+			}
```

Comment:
I'm not sure if you're using the `let e` pattern deliberately or not but [this might be interesting](https://docs.swift.org/swift-book/LanguageGuide/ErrorHandling.html#ID541) in case you didn't already know:
> You write a pattern after catch to indicate what errors that clause can handle. If a catch clause doesn’t have a pattern, the clause matches any error and binds the error to a local constant named error. 

Do you could write instead:
```
} catch {
 	 XCTFail("Error unwrapping database result: \(error)")
}
```

Reply:
Hey, I didn't know about that! Thanks for pointing it out. Is this "magic" variable something that falls within this project's coding guidelines? (It reminds me of synthesized `oldValue` and `newValue` variables in property setters, but neither this `error` nor those variables are referenced in CodingGuidelines.md.)

Reply:
I wondered the same thing. I don't think conversation has been had yet. Maybe we should get it started?

Reply:
It’s totally a-okay to use the magic `error` variable. I do it.

Reply:
Filed as #1512; I'll wait for that to get resolved.